### PR TITLE
Add Remaining text (2 of 2)

### PR DESCRIPTION
**Pull request by jharenza**
❗ staggered on #243 
### Purpose

- Add front matter
- Update intro/discussion

_What GitHub issue does your pull request address?_
#257 
#207 
#206 
#200 
#201 (these are actually in the supplemental.md file and not in the results doc itself)
#259 
#258 

#### Which areas should receive a particularly close look?
<s>immune cell discussion for @pmadsen101  - is this scientifically accurate?</s>
Received feedback from @fosterjb and incorporated.

#### Is there anything that you want to discuss further?
no

#### Is the pull request ready for review?
yes

### Pull review checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [x] All figures follow the [Manubot figure instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#figures)
- [x] There are no new spelling errors identified by the [Manubot spellchecker](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#spellchecking)

### Spellcheck Step

The dictionary used for spellchecking can be updated.
Edit the file in `build/assets/custom-dictionary.txt` by adding new entries to the end.
You do not need to change anything else.
However, if you want to update the first line to have an accurate count of words and you want to remove non-unique ones, run the following command from within `build/assets` on your favorite OS X or Linux machine:
```
(( len = $(awk '!a[$0]++' < custom-dictionary.txt | wc -l ) - 1 )); tmpfile="$(mktemp)"; echo "personal_ws-1.1 en $len utf-8" > $tmpfile; tail -n +2 custom-dictionary.txt | awk '!a[$0]++' >> $tmpfile; mv $tmpfile custom-dictionary.txt
```

### CODE REVIEW

File: content/00.front-matter.md [hunk 1]
```diff
@@ -57,9 +57,20 @@ on {{manubot.date}}.
...
 
 ## Highlights
 <!-- Highlights are bullet points that convey the core findings of your paper. You may include up to four highlights. The length of each highlight cannot exceed 85 characters (including spaces).-->
+OpenPBTA collaborative analyses establish resource for 1,253 pediatric brain tumors
+
+NGS-based WHO-aligned integrated diagnoses generated for 1,260 of 2,840 biospecimens
+
+Classifiers identify medulloblastoma subgroups, TP53 status, and telomerase activity
+
+OpenPBTA will be foundational to accelerate therapeutic translation
```
Comment (sjspielman) [inline content/00.front-matter.md:74]:
```suggestion
OpenPBTA will be foundational to accelerate therapeutic translation and facilitate reproducible genomics in pediatric cancer research 
```

what do you think?

File: content/01.summary.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+With pediatric brain and spinal cord tumors being the leading disease-related cause of death in children, urgently identifying novel, curative therapeutic strategies for these tumors remains a critical need.
+To accelerate this process, the Children's Brain Tumor Network and Pacific Pediatric Neuro-Oncology Consortium have created a systematic process for tumor biobanking, model generation, and sequencing with immediate access to harmonized data.
+Here, we create the OpenPBTA project, in which we openly and collaboratively genomically characterize and molecularly subtype the largest cohort of pediatric brain tumor specimens and histologies to date.
```
Comment (sjspielman) [inline content/01.summary.md:5]: We might want to add a "to our knowledge" phrase here, re-"largest...to date"

File: content/01.summary.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
...
+We establish over 40 scalable analysis modules, demonstrate recapitulation of key oncogenic drivers across histologies, and make the code and data openly available.
```
Comment (sjspielman) [inline content/01.summary.md:6]:
```suggestion
We establish over 40 scalable analysis modules, recapitulate key oncogenic drivers across histologies, and make the code and data openly available.  
```

I think it would be good to add a few more words about the modules, like "over 40 scalable modules leveraging open-source tools to perform common cancer genomics analyses". This is a bit wordy for this sentence, but I think more context about what _are_ these modules would be good.

File: content/04.discussion.md [hunk 4]
```diff
@@ -1,26 +1,31 @@
...
-We created the OpenPBTA project to provide robust code and data resources to the pediatric oncology community, as well as attract collaborators to collaborate on new analyses in order to accelerate therapeutic translation for children with cancer, goals we are seeing play out in real-time.
-To our knowledge, this initiative represents the first large-scale, collaborative open analysis of genomic data coupled with open manuscript writing in which we comprehensively analyzed the largest cohort of pediatric brain tumors to date.
+Through the OpenPBTA, we used an open, reproducible analysis framework within GitHub to bring together researchers and clinicians from across the globe to genomically characterize 1,253 pediatric brain tumors across 59 distinct histologies.
+We created the OpenPBTA project to provide robust code and data resources to the pediatric oncology community, as well as attract interdisciplinary scientists to collaborate on new analyses in order to accelerate therapeutic translation for children with cancer, goals we are seeing play out in real-time.
```
Comment (sjspielman) [inline content/04.discussion.md:4]:
```suggestion
We created the OpenPBTA project to provide robust code and data resources to the pediatric oncology community, as well as to attract interdisciplinary scientists to collaborate on new analyses in order to accelerate therapeutic translation for children with cancer, goals we are seeing play out in real-time.
```

What do we think about "entice" vs "attract"?

File: content/04.discussion.md [hunk 5]
```diff
@@ -1,26 +1,31 @@
...
 
-We applied multiple previously validated classifiers to RNA expression data.
-We predicted medulloblastoma subtypes [@doi:10.1186/s13029-016-0053-y; @doi:10.1371/journal.pcbi.1008263], whether tumors have dysfunctional _TP53_ [@doi:10.1016/j.celrep.2018.03.076], and their degree of telomerase activity [@doi:10.1038/s41467-020-20474-9].
-By assessing _TP53_ and telomerase activity prospectively from expression data, information previously only attainable with genomic sequencing and/or qPCR, we can quickly add knowledge to our existing biological understanding of these tumors.
+We harnessed RNA expression data for a number of analyses, including application of multiple previously validated classifiers.
+From 43/122 (35.2%) medulloblastoma tumors whose pathology reports contained subtype information, we accurately predicted subtypes using MM2S (90.7%) or medulloPackage (95.4%) [@doi:10.1186/s13029-016-0053-y; @doi:10.1371/journal.pcbi.1008263].
```
Comment (sjspielman) [inline content/04.discussion.md:14]:
```suggestion
For example, from 43/122 (35.2%) medulloblastoma tumors whose pathology reports contained subtype information, we accurately predicted subtypes using MM2S (90.7%) or medulloPackage (95.4%) [@doi:10.1186/s13029-016-0053-y; @doi:10.1371/journal.pcbi.1008263].
```

File: content/04.discussion.md [hunk 6]
```diff
@@ -1,26 +1,31 @@
...
+We used additional classifiers to determine whether tumors have dysfunctional _TP53_ [@doi:10.1016/j.celrep.2018.03.076] and their degree of telomerase activity from EXTEND scores [@doi:10.1038/s41467-020-20474-9].
```
Comment (sjspielman) [inline content/04.discussion.md:15]:
```suggestion
We used additional classifiers to determine whether tumors have dysfunctional _TP53_ [@doi:10.1016/j.celrep.2018.03.076] and their degree of telomerase activity from EXTEND scores [@doi:10.1038/s41467-020-20474-9].
```

Should we say "normalized" EXTEND? Not sure.

File: content/04.discussion.md [hunk 7]
```diff
@@ -1,26 +1,31 @@
...
+We found that hypermutant tumors universally display dysregulation of _TP53_ and that EXTEND scores are a robust surrogate measure for telomerase activity, as tumors with high scores have a significant adverse risk for overall survival.
```
Comment (sjspielman) [inline content/04.discussion.md:16]: Single tense:

```suggestion
We found that hypermutant tumors universally displayed dysregulation of _TP53_ and that EXTEND scores were a robust surrogate measure for telomerase activity, such that tumors with high scores had a significant adverse risk for overall survival. 
```

File: content/04.discussion.md [hunk 8]
```diff
@@ -1,26 +1,31 @@
...
+From 43/122 (35.2%) medulloblastoma tumors whose pathology reports contained subtype information, we accurately predicted subtypes using MM2S (90.7%) or medulloPackage (95.4%) [@doi:10.1186/s13029-016-0053-y; @doi:10.1371/journal.pcbi.1008263].
+We used additional classifiers to determine whether tumors have dysfunctional _TP53_ [@doi:10.1016/j.celrep.2018.03.076] and their degree of telomerase activity from EXTEND scores [@doi:10.1038/s41467-020-20474-9].
+We found that hypermutant tumors universally display dysregulation of _TP53_ and that EXTEND scores are a robust surrogate measure for telomerase activity, as tumors with high scores have a significant adverse risk for overall survival. 
+Additionally, high _TP53_ scores are a significant negative prognostic indicator for overall survival for patients with certain tumor types, such as H3 K28-altered DMGs and ependymomas. 
+By assessing _TP53_ and telomerase activity prospectively from expression data, information previously only attainable with DNA sequencing and/or qPCR, we can quickly add oncogenic biomarker and prognostic knowledge to our existing biological understanding of these tumors.
```
Comment (sjspielman) [inline content/04.discussion.md:18]:
```suggestion
By assessing _TP53_ and telomerase activity prospectively from expression data, which is usually only attainable with DNA sequencing and/or qPCR, we incorporated oncogenic biomarker and prognostic knowledge that expanded our biological understanding of these tumors.
```

I'm not sure I followed this sentence. I tried to rephrase to follow it better, but I may or may not have got it right?

File: content/05.administrative.md [hunk 9]
```diff
@@ -21,13 +21,27 @@ CSG's spouse was an employee of Alex's Lemonade Stand Foundation, which was a sp
...
+
+**Figure 3. Mutational co-occurrence and signatures highlight key oncogenic drivers.** 
+A, Bar plot of occurrence and co-occurrence of nonsynonymous mutations for the 50 most commonly mutated genes across all tumor types (annotated from `cancer_group` if N >= 10 or `Other` if N < 10); B, Co-occurrence and mutual exclusivity of nonsynonymous mutations between genes; The co-occurrence score is defined as $I(-\log_{10}(P))$ where $P$ is defined by Fisher's exact test and $I$ is 1 when mutations co-occur more often than expected and -1 when exclusivity is more common; C, The number of SV breaks significantly correlate with CNV breaks (Adjusted R = 0.436, p = 1.08e-37). D, Chromothripsis frequency across pediatric brain tumors shown by `cancer_group` with N >= 3. E, Sina plots of RefSig signature weights for signatures 1, 3, 8, 11, 18, 19, N6, MMR2, and Other across cancer groups. Box plot lines represent the first quartile, median, and third quartile.
```
Comment (sjspielman) [inline content/05.administrative.md:31]:
```suggestion
A, Bar plot of occurrence and co-occurrence of nonsynonymous mutations for the 50 most commonly mutated genes across all tumor types (annotated from `cancer_group` if N >= 10 or `Other` if N < 10); B, Co-occurrence and mutual exclusivity of nonsynonymous mutations between genes; The co-occurrence score is defined as $I(-\log_{10}(P))$ where $P$ is defined by Fisher's exact test, and $I$ is 1 when mutations co-occur more often than expected and -1 when exclusivity is more common; C, The number of SV breaks significantly correlates with the number of CNV breaks (Adjusted $R = 0.436, $P = 1.08\times10^{-37}$). D, Chromothripsis frequency across pediatric brain tumors shown by `cancer_group` with N >= 3. E, Sina plots of RefSig signature weights for signatures 1, 3, 8, 11, 18, 19, N6, MMR2, and Other across cancer groups. Box plot lines represent the first quartile, median, and third quartile.
```

Part of my edit here assumes the LaTeX math will appear for the P-value in the rendering.

File: content/05.administrative.md [hunk 10]
```diff
@@ -21,13 +21,27 @@ CSG's spouse was an employee of Alex's Lemonade Stand Foundation, which was a sp
...
..., Receiver Operating Characteristic for _TP53_ classifier run on FPKM of stranded RNA-Seq samples. B, Violin and box plots of _TP53_ scores plotted by _TP53_ alteration type (N<sub>activated</sub> = 27, N<sub>lost</sub> = 140, N<sub>other</sub> = 999). C, Violin and box plots of _TP53_ RNA expression plotted by _TP53_ activation status (N<sub>activated</sub> = 11, N<sub>lost</sub> = 96, N<sub>other</sub> = 862). D, Box plots of _TP53_ and telomerase (EXTEND) scores grouped by `cancer_group`. Mutation status is highlighted in orange (hypermutant) or red (ultra-hypermutant). E, Heatmap of RefSig mutational signatures for patients who have least one tumor or cell line with a TMB >= 10 Mut/Mb. F, Forest plot depicting the prognostic effects of _TP53_ and telomerase scores on overall survival, controlling for extent of tumor resection, LGG group, and HGG group. G, Forest plot depicting the effect of molecular subtype on overall survival of HGGs. For F and G, hazard ratios (HR) with 95% confidence intervals and p-values are listed. Significant p-values are denoted with black diamonds. Reference groups are denoted by grey diamonds. H, Kaplan-Meier curve of HGG tumors by molecular subtype.
```
Comment (sjspielman) [inline content/05.administrative.md:34]:
```suggestion
A, Receiver Operating Characteristic for _TP53_ classifier run on FPKM of stranded RNA-Seq samples. B, Violin and box plots of _TP53_ scores plotted by _TP53_ alteration type (N<sub>activated</sub> = 27, N<sub>lost</sub> = 140, N<sub>other</sub> = 999). C, Violin and box plots of _TP53_ RNA expression plotted by _TP53_ activation status (N<sub>activated</sub> = 11, N<sub>lost</sub> = 96, N<sub>other</sub> = 862). D, Box plots of _TP53_ and telomerase (EXTEND) scores grouped by `cancer_group`. Mutation status is highlighted in orange (hypermutant) or red (ultra-hypermutant). E, Heatmap of RefSig mutational signatures for patients who have least one tumor or cell line with a TMB >= 10 Mut/Mb. F, Forest plot depicting the prognostic effects of _TP53_ and telomerase scores on overall survival, controlling for category of extent of tumor resection, LGG group status, and HGG group status. G, Forest plot depicting the effect of tumor molecular subtype on overall survival of HGG patients. For F and G, hazard ratios (HR) with 95% confidence intervals and p-values are listed. Significant p-values are denoted with black diamonds. Reference groups are denoted by gray diamonds. H, Kaplan-Meier curve of HGG tumors by molecular subtype.
```

Use American spelling for "gray"

File: build/assets/custom-dictionary.txt [hunk 11]
```diff
@@ -254,6 +259,7 @@ Gottardo
 Governale
 GRCh
 Greiner
+grey
```
Comment (sjspielman) [inline build/assets/custom-dictionary.txt:262]: gray

File: content/05.administrative.md [hunk 12]
```diff
@@ -21,13 +21,27 @@ CSG's spouse was an employee of Alex's Lemonade Stand Foundation, which was a sp
...
+
+**Figure 5. Transcriptomic and Immune Landscape** 
+A, A, First two dimensions from UMAP of sample transcriptome data. Points are colored by `broad_histology` of the samples they represent. B, Heatmap of GSVA scores for Hallmark gene sets with significant differences, with samples ordered by `cancer_group`. C, Box plots of quanTIseq estimates of immune cell proportions in histologies with N > 15. D, Forest plot depicting the effect of immune cell proportion or _CD274_ expression on overall survival of medulloblastomas. Hazard ratio (HR) with 95% confidence intervals and p-values are listed and significant p-values are denoted with colored diamonds. Of note, the upper CI for macrophage M1 HR was too large to plot. E, Box plot of _CD274_ expression (log<sub>2</sub> FPKM) for medulloblastoma samples grouped by molecular subtype. Adjusted p-values are shown.
```
[the remark this answers is not in the document]
Comment (sjspielman) [reply content/05.administrative.md:37]: Replace "proportion" with "fraction" to match text in the figure axis.

File: content/07.supplemental.md [hunk 13]
```diff
@@ -1,24 +1,25 @@
...
 
-**Table S1. Related to Figure 1.** 
+[**Table S1. Related to Figure 1.**](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/tables/results/TableS1-histologies.xlsx)
 Table of specimens and associated metadata, clinical data, and histological data utilized in the OpenPBTA project.
-(Abbreviations: CHDM = chordoma, DMG = diffuse midline glioma, HGG = high-grade glioma, ETMR = embryonal tumor with multilayer rosettes, MB = medulloblastoma, EPN = ependymoma, GNG = ganglioglioma, LGG = low-grade glioma, SEGA = subependymal giant cell astrocytoma, EWS = Ewing sarcoma, CNC = central neurocytoma, EVN = extraventricular neurocytoma, GNT = glial-neuronal tumor, CRANIO = craniopharyngioma)
 
-**Table S2. Related to Figures 2 and 3.** 
+[**Table S2. Related to Figures 2 and 3.**](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/tables/results/TableS2-DNA-results-table.xlsx) 
 Table of TMB, eight CNS mutational signatures, and chromothripsis events per sample.
```
Comment (sjspielman) [inline content/07.supplemental.md:17]:
```suggestion
Excel file with three sheets representing tables of TMB, eight CNS mutational signatures, and chromothripsis events per sample, respectively.
```

File: content/01.summary.md [hunk 14]
```diff
@@ -1,5 +1,10 @@
...
+With pediatric brain and spinal cord tumors being the leading disease-related cause of death in children, urgently identifying novel, curative therapeutic strategies for these tumors remains a critical need.
+To accelerate this process, the Children's Brain Tumor Network and Pacific Pediatric Neuro-Oncology Consortium have created a systematic process for tumor biobanking, model generation, and sequencing with immediate access to harmonized data.
+Here, we create the OpenPBTA project, in which we openly and collaboratively genomically characterize and molecularly subtype the largest cohort of pediatric brain tumor specimens and histologies to date.
+We establish over 40 scalable analysis modules, demonstrate recapitulation of key oncogenic drivers across histologies, and make the code and data openly available.  
+The OpenPBTA has already become a foundation for expansion to other pediatric cancers, has informed molecular tumor board decision-making, and is poised to be an invaluable resource to the pediatric oncology community.
```
[the remark this answers is not in the document]
Comment (jharenza) [reply content/01.summary.md:7]: good point, I can update!

File: content/00.front-matter.md (hunk 1 above)
Comment (sjspielman) [reply content/00.front-matter.md:74]: booooooooo, but fair enough

File: build/assets/custom-dictionary.txt (hunk 11 above)
Comment (jharenza) [reply build/assets/custom-dictionary.txt:262]: this is an either or word with grey as my preferred spelling, but if you have a strong preference for this, we can switch - I won't make judgements (my preferred spelling of this one as well 😄 )

File: content/05.administrative.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (jharenza) [reply content/05.administrative.md:37]: I think you are thinking of _supplemental_ figure 5?

File: content/07.supplemental.md (hunk 13 above)
Comment (jharenza) [reply content/07.supplemental.md:17]: I think we should link these somehow in the manuscript - would you prefer elsewhere or was this something you want to link later? I didn't link specific commits for this because they aren't fully merged.

File: content/01.summary.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (jharenza) [reply content/01.summary.md]: I don't think molecular tumor board needs to be capitalized. made some updates, now at 148/150 words.

File: content/04.discussion.md [hunk 15]
```diff
@@ -1,26 +1,31 @@
 ## Discussion
 
-Through OpenPBTA, we used an open, reproducible analysis framework within GitHub to bring together researchers and clinicians from across the globe to genomically characterize 943 pediatric brain tumors across 59 distinct histologies.
-We created the OpenPBTA project to provide robust code and data resources to the pediatric oncology community, as well as attract collaborators to collaborate on new analyses in order to accelerate therapeutic translation for children with cancer, goals we are seeing play out in real-time.
-To our knowledge, this initiative represents the first large-scale, collaborative open analysis of genomic data coupled with open manuscript writing in which we comprehensively analyzed the largest cohort of pediatric brain tumors to date.
+Through the OpenPBTA, we used an open, reproducible analysis framework within GitHub to bring together researchers and clinicians from across the globe to genomically characterize 1,253 pediatric brain tumors across 59 distinct histologies.
```
[the remark this answers is not in the document]
Comment (jharenza) [reply content/04.discussion.md:3]: updated to "We created the OpenPBTA to provide an open, reproducible analysis framework within GitHub and bring together researchers and clinicians from across the globe to genomically characterize 1,253 pediatric brain tumors across 59 distinct histologies."

File: content/04.discussion.md (hunk 4 above)
Comment (jharenza) [reply content/04.discussion.md:4]: we can go with "entice"

File: content/04.discussion.md (hunk 6 above)
Comment (jharenza) [reply content/04.discussion.md:15]: I actually thought about leaving the term "EXTEND" out of the discussion to make it more readable, so I would suggest no.

File: content/04.discussion.md (hunk 7 above)
Comment (jharenza) [reply content/04.discussion.md:16]: I think we need to keep this active - this is really a call to action to use this data clinically

File: content/04.discussion.md (hunk 8 above)
Comment (jharenza) [reply content/04.discussion.md:18]: yes, but kept active, as we want to relay that this is actively useful to do at diagnosis/time of biopsy

File: content/05.administrative.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (sjspielman) [reply content/05.administrative.md]: Yes, I was!! Good call, all set.

File: build/assets/custom-dictionary.txt (hunk 11 above)
Comment (sjspielman) [reply build/assets/custom-dictionary.txt:262]: Doesn't terribly matter to me!

File: content/04.discussion.md (hunk 6 above)
Comment (sjspielman) [reply content/04.discussion.md:15]: Sounds good. This is more of a methods-y detail anyways.

File: content/00.front-matter.md [hunk 16]
```diff
@@ -57,9 +57,20 @@ on {{manubot.date}}.
...
+<!-- aka the eTOC blurb. A short summary that describes the context and significance of the findings for the broader readership in 80 words or less. -->
+The OpenPBTA is a global, collaborative open-science initiative which brought together researchers and clinicians to genomically characterize over 1,000 pediatric brain tumors. Shapiro, et. al create over 40 open-source, scalable modules to perform cancer genomics analyses and provide a richly-annotated somatic dataset across 59 brain tumor histologies. The OpenPBTA framework can be used as a model for large-scale data integration to inform basic research, therapeutic target identification, and clinical translation. 
 
 ## Highlights
 <!-- Highlights are bullet points that convey the core findings of your paper. You may include up to four highlights. The length of each highlight cannot exceed 85 characters (including spaces).-->
+OpenPBTA collaborative analyses establish resource for 1,253 pediatric brain tumors
+
+NGS-based WHO-aligned integrated diagnoses generated for 1,260 of 2,840 biospecimens
+
+Classifiers identify medulloblastoma subgroups, TP53 status, and telomerase activity
```
Comment (sjspielman) [inline content/00.front-matter.md:72]: I just thought of this - telomerase activity isn't exactly a _classifier_ since its outcome is numeric, right?

File: content/04.discussion.md [hunk 17]
```diff
@@ -1,26 +1,31 @@
...
-Furthermore, in the absence of DNA methylation data, quick molecular subtyping classification could not be performed.
-Thus, to enable more targeted analyses to interrogate the biology of specific tumor subtypes, we created subtyping modules aligned with WHO molecularly-defined diagnoses.
-We worked closely with pathologists and clinicians to build modules from which we determined a research-grade integrated diagnosis for 44% (1260/2840) of biospecimens as well as identify incorrectly diagnosed or mis-identified samples in the OpenPBTA cohort. 
+Furthermore, in the absence of DNA methylation data, rapid classification to derive molecular subtypes could not be performed.
+Thus, to enable biological interrogation of specific tumor subtypes, we created RNA- and DNA-based subtyping modules aligned with WHO molecularly-defined diagnoses.
+We worked closely with pathologists and clinicians to build modules from which we determined a research-grade integrated diagnosis for 44% (1260/2840) of biospecimens as well as discovered incorrectly diagnosed or mis-identified samples in the OpenPBTA cohort.
```
[the remark this answers is not in the document]
Comment (sjspielman) [reply content/04.discussion.md:11]: Noting this is a minor comment, does not need to be incorporated.

File: content/05.administrative.md (hunk 10 above)
Comment (sjspielman) [reply content/05.administrative.md:34]: If we have "grey" in the dictionary, "grey" it is!

File: content/00.front-matter.md (hunk 16 above)
Comment (jharenza) [reply content/00.front-matter.md:72]: tp53 scores are also a numeric outcome, but i suppose that is trained to classify and uses a cutoff (0.5). but looking at the EXTEND paper a bit more, you are right, this is more of an algorithm applied using a gene expression signature, so I will update. thanks!

File: content/05.administrative.md (hunk 9 above)
Comment (jharenza) [reply content/05.administrative.md]: testing

**Comment by jharenza** [comment]
@sjspielman - this is ready for re-review

**Comment by jharenza** [comment]
I am not sure why this is failing...

**Comment by sjspielman** [comment]
@jaclyn-taroni could you have a look here and see if you can figure out why this is failing? I'm not seeing it.

**Comment by jaclyn-taroni** [comment]
Is it the changes to the figure legend introduced in https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/249/commits/42216f21c11d0f8e20b7ec14037cc8376a7bdd37 maybe?

**Comment by jharenza** [comment]
> Is it the changes to the figure legend introduced in [42216f2](https://github.com/AlexsLemonade/OpenPBTA-manuscript/commit/42216f21c11d0f8e20b7ec14037cc8376a7bdd37) maybe?

hmm, ahh that was the LaTeX test from @sjspielman :P - let me take that out and try again

**Comment by jaclyn-taroni** [comment]
Tried something with https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/249/commits/9507cab5626de5b212bc174640b1cd52b21c90e6

**Review by sjspielman — APPROVED** [review]
🎉
